diff -Naur php-7.3.4_old/sapi/fpm/init.d.php-fpm.in php-7.3.4/sapi/fpm/init.d.php-fpm.in
--- php-7.3.4_old/sapi/fpm/init.d.php-fpm.in	2020-06-15 20:10:33.212252162 +0200
+++ php-7.3.4/sapi/fpm/init.d.php-fpm.in	2020-06-15 20:10:01.564952133 +0200
@@ -18,7 +18,7 @@
 php_fpm_PID=@localstatedir@/run/php-fpm.pid
 
 
-php_opts="--fpm-config $php_fpm_CONF --pid $php_fpm_PID"
+php_opts="--fpm-config $php_fpm_CONF --pid $php_fpm_PID -R"
 
 
 wait_for_pid () {
