From 28583edfd5245a2ce288f6eb746f3b921d732310 Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@open-nandra.com>
Date: Thu, 11 Jun 2020 08:49:49 +0200
Subject: [PATCH] Makefile fixes

Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>
---
 Makefile | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 85cfc99..1990aa2 100644
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ deps:
 qa: lint cs
 
 install:
-	install -d -o www-data $(CACHE_DIR)
+	install -d $(CACHE_DIR)
 	install -d $(CONFIG_DIR)
 	cp app/config/* $(CONFIG_DIR)
 	install -d $(CONFIG_DIR)/certs
@@ -66,8 +66,8 @@ install:
 	cp version.json $(DATA_DIR)
 	cp -r vendor/ $(DATA_DIR)
 	cp -r www/ $(DATA_DIR)
-	install -d -o www-data $(LOG_DIR)
-	install -d -o www-data ${DESTDIR}/var/lib/iqrf-gateway-webapp
+	install -d $(LOG_DIR)
+	install -d ${DESTDIR}/var/lib/iqrf-gateway-webapp
 	# Delete documentation
 	find ${VENDOR_DIR} -type f -name "AUTHORS*" -delete
 	find ${VENDOR_DIR} -type f -name "CHANGELOG*" -delete
@@ -141,3 +141,5 @@ test: deps
 
 webpack:
 	webpack --mode production
+clean:
+	true
-- 
2.7.4

